### PR TITLE
4432 new mission

### PR DESCRIPTION
**Pull request by robert-bryson**
related to https://github.com/GSA/data.gov/issues/4432

### CODE REVIEW

**Comment by btylerburton** [comment]
Good first pass, Robbie. Here's some feedback I see straight away.

Home page:
- The "About Data.gov" block should be removed from the home page
- Mission section column width should be adjusted to match comps
- Background image should be scaled appropriately and alpha value adjusted to match comps

About page:
- Remove links at top of page
- Add missing two column layout changes
- Add border for media inquiries box

**Comment by robert-bryson** [comment]
Well, I can't approve my own PR, but the changes look great! Thanks @btylerburton!

**Review by tdlowden — COMMENTED** [review]
i noticed in the `About Us` section, the background image and separation in the Figma wireframes doesn't appear. Maybe intentional? The image does look a bit pixelated in figma:

image


Also in `About Us`, to @FuhuXia's point in the chat the other day, the (very cool) image's text is a bit small, esp with the margins that shrink everything a bit more than the figma renderings. Almost feels like that graphic could span the width of the page on its own

image


Made one small comment on line 50 of pages/about.html regarding pluralizing "Technology Transformation Services"

File: pages/about.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        is, please <a href="{% page '/contact/'%}">contact us</a>.
+
+      </div>
+      <div class="grid-col-12 desktop:grid-col-6 mobile-top-padding">
+        <b>Federal agencies are required to:</b>
+        <ul>
+          <li>Create a Single Agency Data Inventory.** Agencies are required to catalog their data assets, just like
+            they would inventory computers or desk chairs, to better manage and use these resources.</li>
+          <li>Publish a Public Data Listing.** Agencies are required to publish a list of their data assets that are
+            public, or could be made public. This list is made available as a data.json hosted at the primary domain
```
Comment (nickumia) [inline pages/about.html:94]:
```suggestion
          <li><b>Create a Single Agency Data Inventory.</b> Agencies are required to catalog their data assets, just like
            they would inventory computers or desk chairs, to better manage and use these resources.</li>
          <li><b>Publish a Public Data Listing.</b> Agencies are required to publish a list of their data assets that are
            public, or could be made public. This list is made available as a data.json hosted at the primary domain
```

I wasn't sure if the `**` was pointing to an endnote or was supposed to be part of bolding or a separator element... but the most logical answer is to just remove it.

File: pages/about.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  <div class="usa-prose">
+    <h1 id="about">{{ title }}</h1>
+  </div>
+  <section class="usa-section">
+    <div class="grid-row grid-gap-lg">
+      <div class="grid-col-12 desktop:grid-col-5 ">
+        <h4 id="mission">Mission</h4>
+        <div>
+          The United States Government's open data site is designed to unleash the power of government open data to
+          <b>inform decisions by the public and policymakers, drive innovation and economic activity, achieve agency
```
Comment (nickumia) [inline pages/about.html:24]:
```suggestion
          <b>inform decisions of the public and policymakers, drive innovation and economic activity, achieve agency
```
I would suggest either "of the public" or "made by the public".  "by the public" seems a bit too ambiguous.

File: pages/index.html [hunk 3]
```diff
@@ -46,38 +40,36 @@ <h4 class="text-uppercase">
...
-        <h1>About Data.gov</h1>
+      <div class="grid-col-12 desktop:grid-col-12 mission__copy">
+        <h1>Mission</h1>
         <div>
-          Data.gov is the United States government's open data website. It provides access to datasets published by agencies
-          across the federal government. Data.gov is intended to provide access to government open data to the public, achieve
-          agency missions, drive innovation, fuel economic activity, and uphold the ideals of an open and transparent government.
+          The United States Government’s open data site is designed to unleash the power of government open data to
+          <b>inform
+            decisions by the public and policymakers</b>, <b>drive innovation and economic activity</b>, <b>achieve
```
Comment (nickumia) [inline pages/index.html:66]:
```suggestion
            decisions of the public and policymakers</b>, <b>drive innovation and economic activity</b>, <b>achieve
```
Same as above.

File: pages/about.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            target="_blank" class="usa-link usa-link--external">Technology Transformation Services</a>.
+        </div>
+      </div>
+      <div class="grid-col-12 desktop:grid-col-6 mobile-top-padding">
+        <h2 id="technology">With what technology is Data.gov built?</h2>
+        <div>
+          Data.gov is powered by two open source applications, <a href="https://ckan.org" target="_blank"
+            class="usa-link usa-link--external">CKAN</a> and
+          <a href="https://www.11ty.dev/" target="_blank" class="usa-link usa-link--external">11ty</a> and it is
+          developed publicly on <a href="https://github.com/GSA/catalog-deploy" target="_blank"
```
Comment (nickumia) [inline pages/about.html:59]:
```suggestion
          developed publicly on <a href="https://github.com/GSA/data.gov" target="_blank"
```

**Review by nickumia — COMMENTED** [review]
I think the new page is very informative and has a nicely integrated decision.  I love the work put into this ❤️

**Comment by btylerburton** [comment]
Converting this to a draft until I can put in the required changes.

**Comment by btylerburton** [comment]
This PR is ready for another review.

File: pages/index.html (hunk 3 above)
Comment (btylerburton) [reply pages/index.html:69]: I'm gonna leave this as is. It reads fine.

File: pages/about.html (hunk 2 above)
Comment (btylerburton) [reply pages/about.html:24]: Clear enough as is.

**Review by FuhuXia — APPROVED** [review]

**Comment by FuhuXia** [comment]
LGTM

**Pull request merged**